common/src/main/java/com/movie/recommender/common/model/movie/MovieCredits.java

```diff
@@ -1,26 +1,66 @@
 package com.movie.recommender.common.model.movie;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 import java.util.List;
 
 @Data
 public class MovieCredits {
     private int id;
     private List<CastMember> cast;
+    private List<CrewMember> crew;
 
     @Data
     public static class CastMember {
+        private int id;
         private boolean adult;
+        private String character;
+
+        @JsonProperty("cast_id")
```

Comment:
configure `ObjectMapper` with `PropertyNamingStrategies.SNAKE_CASE` to handle snake_case to camelCase mapping automatically, which will remove the need for `@JsonProperty` annotations on each field